<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ErrWindow</class>
 <widget class="QDialog" name="ErrWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>330</width>
    <height>146</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ошибка</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>261</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Все поля должны быть заполнены</string>
   </property>
  </widget>
  <widget class="QPushButton" name="errButton">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>100</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>ОК</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
